web_src/js/features/imagediff.js

```diff
@@ -126,13 +128,24 @@ export function initImageDiff() {
 
       const widthChanged = sizes.$image1.length !== 0 && sizes.$image2.length !== 0 && sizes.$image1[0].naturalWidth !== sizes.$image2[0].naturalWidth;
       const heightChanged = sizes.$image1.length !== 0 && sizes.$image2.length !== 0 && sizes.$image1[0].naturalHeight !== sizes.$image2[0].naturalHeight;
-      if (sizes.$image1.length !== 0) {
-        $container.find('.bounds-info-after .bounds-info-width').text(`${sizes.$image1[0].naturalWidth}px`).addClass(widthChanged ? 'green' : '');
-        $container.find('.bounds-info-after .bounds-info-height').text(`${sizes.$image1[0].naturalHeight}px`).addClass(heightChanged ? 'green' : '');
+      if (sizes.$image1 && sizes.$image1.length !== 0) {
+        const boundsInfoAfterWidth = this.querySelector('.bounds-info-after .bounds-info-width');
+        boundsInfoAfterWidth.textContent = `${sizes.$image1[0].naturalWidth}px`;
+        boundsInfoAfterWidth.classList.add(widthChanged ? 'green' : '');
+
+        const boundsInfoAfterHeight = this.querySelector('.bounds-info-after .bounds-info-height');
+        boundsInfoAfterHeight.textContent = `${sizes.$image1[0].naturalHeight}px`;
+        boundsInfoAfterHeight.classList.add(heightChanged ? 'green' : '');
```

Comment:
It doesn't seem right.

```
temp1.classList.add('')
VM182:1 Uncaught DOMException: Failed to execute 'add' on 'DOMTokenList': The token provided must not be empty.
    at <anonymous>:1:17
```

Reply:
[`69982e1` (#30140)](https://github.com/go-gitea/gitea/pull/30140/commits/69982e1385c828db0c27f23e880fe4d5648ed9f5)